coldfront/core/utils/mou.py

```diff
@@ -0,0 +1,182 @@
+from coldfront.core.utils.common import import_from_settings
+from coldfront.core.resource.utils_.allowance_utils.computing_allowance import ComputingAllowance
+import datetime
+
+def upload_to_func(instance, filename):
+    fs = import_from_settings('FILE_STORAGE')
+    path = ''
+    type_ = ''
+    from coldfront.core.allocation.models import (AllocationAdditionRequest,
```

Comment:
Any reason to put these imports within the function?

Reply:
to avoid circular imports